src/Apropos/LogicalModel/Enumerable.hs

```diff
@@ -4,3 +4,5 @@ module Apropos.LogicalModel.Enumerable (
 
 class (Eq p, Ord p) => Enumerable p where
   enumerated :: [p]
+  default enumerated :: (Enum p, Bounded p) => [p]
+  enumerated = [minBound .. maxBound]
```

Comment:
Somehow I have not come across this syntax before. It's quite nice. :)

Reply:
Is the `Enumerable` here equivalent to [`Finite`](https://hackage.haskell.org/package/universe-base-1.1.3/docs/Data-Universe-Class.html#t:Finite)?